_posts/0000-01-02-bhavyasy.md

```diff
@@ -0,0 +1,5 @@
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
+Use the left arrow to go back!
```

Comment:
I have learned Git through this interactive course. 
This is a list of things that I had learned.
- [x]   What is Git and basic command.

- [x]  haw to pull and push.

- [x]  What is branch and haw to change the file title.